mojo-trifacta-udf/src/main/java/ai/h2o/dai/trifacta/mojo/udf/H2oMojoScorerLocal.java

```
@@ -0,0 +1,70 @@
+package ai.h2o.dai.trifacta.mojo.udf;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.List;
+
+import common.ai.h2o.dai.mojo.MojoScoringInterface;
+import ai.h2o.mojos.runtime.frame.MojoFrameMeta;
+import com.trifacta.trifactaudfs.TrifactaUDF;
+import ai.h2o.mojos.runtime.MojoPipeline;
+import ai.h2o.mojos.runtime.lic.LicenseException;
+import com.google.gson.reflect.TypeToken;
+import com.google.gson.Gson;
+
+
+public class H2oMojoScorerLocal implements TrifactaUDF<String> {
+
+    private MojoPipeline _MOJO;
+    private MojoFrameMeta _MOJOFRAMEMETA;
+    private String _DELIMITER;
+    private String _OUTPUT;
+    private boolean _error;
+
+    @Override
+    public String exec(List<Object> inputs) throws IOException {
+        if (inputs == null) {
```

Does this happen? Should we rather throw an exception?